src/main/kotlin/no/nav/helse/k9format/K9Format.kt

```diff
@@ -0,0 +1,225 @@
+package no.nav.helse.k9format
+
+import com.fasterxml.jackson.databind.JsonNode
+import no.nav.helse.prosessering.v1.*
+import no.nav.helse.prosessering.v1.Tilsynsordning
+import no.nav.k9.søknad.JsonUtils.getObjectMapper
+import no.nav.k9.søknad.felles.*
+import no.nav.k9.søknad.felles.Barn
+import no.nav.k9.søknad.pleiepengerbarn.*
+import no.nav.k9.søknad.pleiepengerbarn.Beredskap
+import no.nav.k9.søknad.pleiepengerbarn.Utenlandsopphold
+import java.lang.IllegalArgumentException
+import java.math.BigDecimal
+import java.time.LocalDate
+
+fun PreprossesertMeldingV1.tilK9PleiepengeBarnSøknad(): JsonNode {
+    val språk = when (sprak) {
+        "nb" -> Språk.NORSK_BOKMÅL
+        "nn" -> Språk.NORSK_NYNORSK
+        else -> Språk.NORSK_BOKMÅL
+    }
+    val builder = PleiepengerBarnSøknad.builder()
+        .søknadId(SøknadId.of(soknadId))
+        .mottattDato(mottatt)
+        .språk(språk)
+        .søker(soker.tilK9Søker())
+        .arbeid(
+            arbeidsgivere.tilK9Arbeid(
+                frilans,
+                selvstendigVirksomheter,
+                Periode.builder().fraOgMed(fraOgMed).tilOgMed(tilOgMed).build()
+            )
+        )
+        .barn(barn.tilK9Barn())
+        .søknadsperiode(Periode.builder().fraOgMed(fraOgMed).tilOgMed(tilOgMed).build(), SøknadsperiodeInfo())
+
+    builder.bosteder(medlemskap.tilK9bosteder())
+
+    beredskap?.let {
+        builder.beredskap(
+            Beredskap.builder().periode(
+                Periode.builder().fraOgMed(fraOgMed).tilOgMed(tilOgMed).build(),
+                Beredskap.BeredskapPeriodeInfo.builder().tilleggsinformasjon(beredskap.tilleggsinformasjon).build()
+            ).build()
+        )
+    }
+
+    utenlandsoppholdIPerioden?.let {
+        builder.utenlandsopphold(utenlandsoppholdIPerioden.tilK9Utenlandsopphold())
+    }
+
+    nattevaak?.let {
+        builder.nattevåk(
+            Nattevåk.builder()
+                .periode(
+                    Periode.builder().fraOgMed(fraOgMed).tilOgMed(tilOgMed).build(),
+                    Nattevåk.NattevåkPeriodeInfo.builder().tilleggsinformasjon(it.tilleggsinformasjon ?: "").build()
```

Comment:
Helst ikke send in tom string som default, er det noe i k9-format som forcer at man må gjøre det?

Reply:
Nei, egentlig ikke. Fikser det nå :)